@here/harp-examples/src/datasource_features_polygons.ts

```diff
@@ -281,7 +281,7 @@ export namespace PolygonsFeaturesExample {
         const mapView = new MapView({
             canvas,
             projection: sphereProjection,
-            clipPlanesEvaluator: new TopViewClipPlanesEvaluator(300000),
+            clipPlanesEvaluator: new TiltViewClipPlanesEvaluator(),
```

Comment:
Good point, since globe controls have now tilt added, somebody should update this too.

Reply:
Please remove this and make TiltViewClipPlanesEvaluator the default one

Reply:
I think it should be removed in the patch-set that changes the TiltViewClipPlanesEvaluator to default one. We need to firstly merge this one, in order not to break tests, etc....

Reply:
TiltViewClipPlanesEvaluator is now default as part of this PR.
This should make #922 obsolete.